NameLayerMain/src/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -205,6 +207,18 @@ public void run() {
 			ver = updateVersion(ver, plugin.getName());
 			log(Level.INFO, "Database update to Version six took " + (System.currentTimeMillis() - first_time) /1000 + " seconds.");
 		}
+		if (ver == 6){
+			long first_time = System.currentTimeMillis();
+			log(Level.INFO, "Database upadting to version seven.");
+			db.execute("create table if not exists group_invitation(" + 
+					"uuid varchar(40) NOT NULL," +
```

Comment:
Should just be a char field since the UUID will always be 36 characters.

On Thu, Jan 14, 2016 at 10:36 AM, rourke750 <notifications@github.com>
wrote:

> In
> NameLayerMain/src/vg/civcraft/mc/namelayer/database/GroupManagerDao.java
> <https://github.com/Civcraft/NameLayer/pull/154#discussion_r49765039>:
>
> > @@ -205,6 +207,18 @@ public void run() {
> >  			ver = updateVersion(ver, plugin.getName());
> >  			log(Level.INFO, "Database update to Version six took " + (System.currentTimeMillis() - first_time) /1000 + " seconds.");
> >  		}
> > +		if (ver == 6){
> > +			long first_time = System.currentTimeMillis();
> > +			log(Level.INFO, "Database upadting to version seven.");
> > +			db.execute("create table if not exists group_invitation(" +
> > +					"uuid varchar(40) NOT NULL," +
>
> Yea.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Civcraft/NameLayer/pull/154/files#r49765039>.
>